contrib/gardener/manifests/rbac-cp.yml

```diff
@@ -84,37 +84,7 @@ roleRef:
 
 ---
 
-# Define role for OPA/kube-mgmt to update configmaps with policy status.
-kind: Role
-apiVersion: rbac.authorization.k8s.io/v1
-metadata:
-  namespace: opa
-  name: configmap-modifier
-rules:
-- apiGroups: [""]
-  resources: ["configmaps"]
-  verbs: ["update", "patch"]
-
----
-
-# Grant OPA/kube-mgmt role defined above.
-kind: RoleBinding
-apiVersion: rbac.authorization.k8s.io/v1
-metadata:
-  namespace: opa
-  name: opa-configmap-modifier
-subjects:
-- kind: User
-  name: system:kubemgmt
-  apiGroup: rbac.authorization.k8s.io
-roleRef:
-  kind: Role
-  name: configmap-modifier
-  apiGroup: rbac.authorization.k8s.io
-
----
-
-# Define role for OPA/kube-mgmt to cache the resources
+# Define role for kube-mgmt to cache the resources
```

Comment:
I think this kube-mgmt stuff was only related to OPA and, thus, can be deleted as well.

Reply:
This point is still open.

Reply:
Should be deleted (see 325d71e8cd0e6c48ea81e1ebd7b16b400afa8889)